Extensions - Remove Details from Documentation (#218)
rancher-sandbox/docs.rancherdesktop.io: docs/how-to-guides/installing-uninstalling-extensions.md

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```

#### Using the UI

> Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.

Reviewer: ```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```
